lib/stdlib/doc/src/orddict.xml

```diff
@@ -38,7 +38,9 @@
     <p>This module provides a <c>Key</c>-<c>Value</c> dictionary.
       An <c>orddict</c> is a representation of a dictionary, where a
       list of pairs is used to store the keys and values. The list is
-      ordered after the keys in the <em>Erlang term order</em>.</p>
+      ordered after the keys in the <em>Erlang term order</em>:</p>
+      <pre>
+number &lt; atom &lt; reference &lt; fun &lt; port &lt; pid &lt; tuple &lt; map &lt; nil &lt; list &lt; bit string</pre>
```

Comment:
It seems unnecessary to list all types here. I think it is better to turn the text **Erlang term order** into a link to the definition of term order.